filament/backend/src/DriverBase.h

```diff
@@ -75,6 +75,7 @@ struct HwBufferObject : public HwBase {
 struct HwIndexBuffer : public HwBase {
     uint32_t count : 27;
     uint32_t elementSize : 5;
+    uint8_t bufferVersion{};      
```

Comment:
nit: `bufferObjectVersion`? If I get it right, it's used for BOs only, and it serves the same purpose as `HwVertexBuffer::bufferObjectsVersion`.

Reply:
fixed in https://github.com/shapr3d/filament/pull/89/commits/19d96e5114fbf8884f3556634a95fd3018a938ab